pkg/webui/console/store/actions/gateway.js

```diff
@@ -32,52 +25,67 @@ import {
   createStopEventsStreamActionType,
   clearEvents,
   createClearEventsActionType,
-} from '../actions/events'
+} from './events'
 
-import {
-  getApiKey,
-  createGetApiKeyActionType,
-  getApiKeySuccess,
-  createGetApiKeySuccessActionType,
-  getApiKeyFailure,
-  createGetApiKeyFailureActionType,
-} from './api-key'
+import { createGetApiKeyActionType } from './api-key'
+
+import { createRequestActions } from './lib'
 
 export const SHARED_NAME = 'GATEWAY'
 
-export const GET_GTW = 'GET_GATEWAY'
-export const GET_GTW_SUCCESS = 'GET_GATEWAY_SUCCESS'
-export const GET_GTW_FAILURE = 'GET_GATEWAY_FAILURE'
+export const GET_GTW_BASE = 'GET_GATEWAY'
+export const [{
+  request: GET_GTW,
+  success: GET_GTW_SUCCESS,
+  failure: GET_GTW_FAILURE,
+}, {
+  request: getGateway,
+  success: getGatewaySuccess,
+  failure: getGatewayFailure,
+}] = createRequestActions(GET_GTW_BASE, (id, meta) => ({ id, meta }))
```

Comment:
this will add `meta` to the `payload` object entry.
<img width="381" alt="Screenshot 2019-06-13 at 10 08 56" src="https://user-images.githubusercontent.com/16374166/59415227-5bcfe080-8dc3-11e9-939b-077f6ee4f340.png">
please check how it is done in `redux-actions`